### ISSUE TITLE
context menu toggle to allow ads on a site

### ISSUE THREAD

**Issue by tapper82** [gorhill/uBlock#1975]
Read first: <https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md>

### One or more specific URLs where the issue occurs

[URL(s) for issue on a specific site are **mandatory**]

### Screenshot in which the issue can be seen

[Screenshot(s) for visual issues are **mandatory**]

### Steps for anyone to reproduce the issue

1. Be as detailed as possible:
1. Because we are not looking over your shoulder and
1. Keep in mind: Nobody can read your mind.

### Your settings

[If you fail to provide this info, I will mark the issue as invalid. Lists all settings which differs from default settings]

- Browser/version: 
- uBlock Origin version: 

##### Your filter lists

[Example: "Default filter lists + FRA", or "Default filter lists minus uBlock -- Badware risks"]

##### Your custom filters (if any)
Hi I use a screen reader and cant allow ads for a site I want to support such as http://www.phoronix.com/scan.php?page=home

I am using firefox 51 and want to use the applications key to bring up the context menu to allow ads for a site. I cant click in the toolbar as i do not use the mouse thanks

**Comment by anewuser** [comment]
You've ignored and deleted the template text that appears when you create a new issue. That text isn't there just for decoration. 

Use this page only to report bugs, i.e. problems that make uBlock Origin stop working. Please close this issue and read this carefully before posting here again: [CONTRIBUTING.md](https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md).

> For support/discussions, there is [Mozilla Discourse](https://discourse.mozilla-community.org/t/support-ublock-origin/6746).

**Comment by tapper82** [comment]
Hi I use a screen reader! Wen i posted this bug my screen reader dropped me in to the edit box and i just started typing so i don't know any thing about any template text. I can't give you screen shots as i cant see the screen thanks for being so much help. Pece and love Tapper. :(

**Comment by anewuser** [comment]
I'm sorry. I misread the part where you said "screen reader" before. I wasn't aware of your needs and challenges.

Since this is a matter of usability for people who use screen readers, this is a valid issue.

**Comment by Atavic** [comment]
Hi Tapper, Phoronix website uses the top level host: static.hvvc.us
So it should be enabled on uMatrix filters.

**Comment by pzajda** [comment]
Hi,

To be sure, should we describe a little more the issue.
For now, when opening context menu using keyboard (pressing the Application key), we have the ability to block an element.
Nothing to unblock.
Nor any menu to allow or disallow everything on a domain.

Because we cannot access easily to tool bars menus, having these ability would largely increase accessibility of Ublock Origin.
Another possibility could be to add a "Ublock Origin" submenu in the Tools menu, in the Firefox menu bar.
For information, we have not this issue in Adblock. We have adblock menu on the "Tool" firefox menu (in the menu bar) which is a good approach because menu bar is accessible.

Thanks.
